### ISSUE TITLE
Decompiled source code for this app in case you think is bad for your PC

### ISSUE THREAD

**Issue by LockNessMonster1** [goatfungus/NMSSaveEditor#1118]
https://file.io/MgN2921QHhpL

[nmseditorcode.zip](https://github.com/user-attachments/files/17670201/nmseditorcode.zip)

**Comment by LockNessMonster1** [comment]
It will need some programmer work, but then, I guess there are lots of them out there using this... :D

**Comment by LockNessMonster1** [comment]
You should not reject the help from anyone who is willing to offer it's help dude, that's bad mojo... and delete these posts as you like, it just shows how untrusty you are, if you have to hide criticism then there's something wrong with you

**Comment by LockNessMonster1** [comment]
and I can just came here every week and repost again win the same ZIP file or an updated one

**Comment by LockNessMonster1** [comment]
apologize and move on

**Comment by LockNessMonster1** [comment]
Related with #1108 and other issue who was shamelessly deleted to hide the owner of this repository shame

**Comment by LockNessMonster1** [comment]
[Uploading nmseditorcode.zip…]()

**Comment by LockNessMonster1** [comment]
https://file.io/zwuXMkkwnJzR

**Comment by LockNessMonster1** [comment]
just a refresh, just because I can :D

**Comment by LockNessMonster1** [comment]
please dont't forget to check #1108

**Comment by The-Great-Sephiroth** [comment]
You're kind of coming across as aggressive here. Goat is probably busy with life. Some of us work long days. I have a 9PM in an hour. I work in IT. I started at 7:15 this morning. Just life. He'll update it when he can. HG changed the save format. He'll figure it out.

**Comment by LockNessMonster1** [comment]
@The-Great-Sephiroth I'm sure he's got a life, that partially the point of all of this. You surely have not have been following this soap opera from the beginning so let me put you up to date:

There was this user who created an issue here (the one who got deleted) suggesting to @goatfungus to release it's code as open source since this way it would be easier to get help and he would have to spend less time worrying with this except by reviewing PR's.

That post was ignored and the original author of the post just closed the issue him self.

Later in the issue mentioned in this one the author of that issue spoke his mind and the response of @goatfungus  was to go to the original issue (which was already closed) and accuse the author of trying to **steal his work** just for offering his help!!!

@goatfungus  deleted that issue but not before I've read it, and then deleted all the conversation with the other participant in question trying to make him self some kind of super hero I guess.

This post is to make a point. The app on this repo is Java, there's dozens of decompilers on the web that can decompile the jar in this app, the code above is extracted by one of those freely available tools. If someone wanted to steal the code it wouldn't even had to ask!!! @goatfungus move was a dick move!

If he does not need help it's is prerogative but in no scenario he has the right of accusing anyone else of stealing without evidence.

Now the code is visible for everyone and @goatfungus is free to delete this post also, the links in here are 24h only.

I just couldn't stand here and do nothing while witnessing this level of injustice. 

I'm not coming across as aggressive here, I'm just making sure that @goatfungus realizes that he he did a dick move and should have handled things differently. After all this is his repository and his app but that does not give him the right to disrespect other people or creating a scenario where they look like villains!

**Comment by LockNessMonster1** [comment]
> You're kind of coming across as aggressive here. Goat is probably busy with life. Some of us work long days. I have a 9PM in an hour. I work in IT. I started at 7:15 this morning. Just life. He'll update it when he can. HG changed the save format. He'll figure it out.

BTW @The-Great-Sephiroth I'm on IT also. that's why I' taking this so personally. You started at 7:15 this morning? I start every day at 7 a.m.!!! and end up most of the days after 21 p.m. which is the the time I finally got to relax a bit, today is a bit later, 23H or so.... that's why I appreciate so much when the community drops in and offers help.

Most of us have families, limited time, and that some of us offer the time to help and relief the charge and stress on someone that contributes to us all freely is a blessing!

But there's a line, THOSE WHO GIVE FREELY SHOULD NOT HAVE TO BE THREATEN AS CRIMINALS! And when someone offers to help, either you accept or kindly refuse, but YOU DO NOT INSULT OR IN ANY OTHER WAY OFFEND THAT PERSON!

That's not how communities work. I personally contribute to several open source projects ans it's gratifying, I fell realized and happy to be part of those communities.

But what I've witnessed here has nothing to do with that, it's just this "demi-god" who thinks he can do what he pleases because he has this code that he thinks no one else can touch... and he just threats the people o try to help him as shit... and dam, that's wrong!

@goatfungus you don't want help? fine! But don't put yourself above and beyond the people that is trying to contribute to your work, and surely accusing them of stealing is surely not the way... worst, deleting posts and messages just proves that you're hiding stuff!!! Dah!!!

You're not deleting they're messages and posts, you're deleting YOU'RE reaction! And if you cannot handle the criticism and hide yourself behind deleted messages than you're the one with something to hide.

Be transparent, accept criticism ans reply presenting arguments refuting what you think is wrong!

Every strong man faces it's enemies!!!

**Comment by emilio-simoes** [comment]
Thank you @LockNessMonster1 , I was actually not expecting this kind of reaction from no one and I'm so happy to realize that I'm not the only one seeing things this way!

@The-Great-Sephiroth please don't judge but @LockNessMonster1 does touches all the relevant points and all that is wrong with this repo!

Once you are ready to share at this level you have to be prepared that you will not have the time to be available every day, that's why you need a community around you. Just like the NMS community made that game great this project needs a community to make it equally great, and @goatfungus can not do it it alone!

Either he realizes it and gives something to the community that can be used is the future (maybe in Light No Fire) or he just keeps this to himself as a selfish dude... he will not be making any money from this as the IP of the game belongs to Hello Games so he would be surely happy with the help from the community!!! That's what's this game as always been about!

Just as a final notice! All the IP of the game has always been owned by Hello Games so any activity in this repo as been kind of dubious... but I'm not getting into that and I'll just let you guys play freely. (but keep that in mind) 
Anyhow... happy thoughts and merry Christmas to you all!!! (I know, it's just November and every one is all around it already ... sad world we are living these days)

**Comment by LockNessMonster1** [comment]
@The-Great-Sephiroth you clearly should not make statements without knowing the other participants reality...

**Comment by The-Great-Sephiroth** [comment]
This is not the place for this. You're acting entitled. This is his save editor. He wrote it. If he wants help he'll ask for it. What happened to owning something you built? It's not a matter of knowing anything. You're simply frustrated that the guy won't accept help on HIS PROJECT and that he won't open-source something he poured his time into.

I'm a C++ programmer and have been since the 90's. I have a LOT of code I have never shared. People loved my apps, but I kept the code to myself. Not everything is open-source, and I'm a Gentoo user. No sense getting all flustered because you can't get something you want though.

**Comment by emilio-simoes** [comment]
@The-Great-Sephiroth so what you're saying is that you are a greedy and selfish guy? I'm really sorry, that's what not really my intention to make people reveal their darkest secrets! Like you being a C++ developer!!! Man! that must be really hurtful to you! I'm so sorry for you!

But I guess that's what you take when you are used to "keep you're creations" to yourself, you end up believing you are better than the rest of us!

Open source developers don't suffer from that stigma because they share every day of they lives!

I there is frustration here is not from the lack of sharing, is from the lack of vision.

And @The-Great-Sephiroth , I'm so sorry you are a C++ developer since the 90's, I can't imagine how many nights you have spend crying in your pillow every night knowing the nightmares you have faced the day before, it must have been horrible!!! I'm happy some nice guys finally took pity on C++ and made it a usable language since 2011, which probably was to late for you since your brain by that time would be a pulp... but hey, you can always try Rust, I'm sure it will fix most of the trauma!

**Comment by emilio-simoes** [comment]
@LockNessMonster1 you know what it's best in is dialog? Is this guy coming here claiming hes a Linux contributor with close source software which just goes around and breaks all the GNU Linux philosophy 😂

**Comment by LockNessMonster1** [comment]
At least he claims to be using Gentoo which would make it really hard for it to write any code 🤭

**Comment by emilio-simoes** [comment]
We could try to ask for prof but since it's all closed source he wouldn't be able to present any prof without revealing himself 😂

**Comment by The-Great-Sephiroth** [comment]
Wow, you guys sure seem to lack common-sense and basic understanding. I never said I was some kind of active contributor to anything. I simply said it's a person's right to do with their code what they desire. Since you seem to be on the other side of the fence, how about I come over and just borrow you car? I mean I can take it paint it a new color because why not? Who cares that you bought it, got it just the way you wanted it, and then somebody else feels entitled to it.

You guys clearly have no clue what you're talking about and are simply trolls. Nice assumptions on everything. Just because you're so stupid you cannot pick up C++ doesn't mean it's difficult or a nightmare of any kind. It just means you're not cut out to handle a language that doesn't require a thousand libraries to function on it's own. But hey, anybody can learn Basic or .NET and not have to worry about a thing.

**Comment by LockNessMonster1** [comment]
Yeah! @The-Great-Sephiroth I guess your missing the point here! It's not about taking, it's about sharing! And you don't know a shit about sharing, from what you have told you spent you're entire life feeding on other people knowledge, absorbing and sucking without giving back! You're a parasite!

For your information I'm a proficient C++ developer which as made contributions to dozens of C++ projects mostly on the C++11 and c++14 versions of the language! Just as in C++ I contributed in projects with languages like TypeScript, Java, C# and JavaScript! I'm an active member of the community which **spends some of my free time giving to others!!!!** And you, what do you have to say from yourself??? 

You call us trolls? You might just be the biggest troll here because you can't even take a joke, you're a sad person who just need to make other people feel sad so that you feel good!!! But let me give you some news, you're just sad and pathetic as any one of us, maybe even worst because you just don't let this go!!! This is an insignificant thread and you seem to be the only one who cares about it...

**Comment by laLocaLuka** [comment]
![image](https://github.com/user-attachments/assets/b83119d4-14b5-40fd-86b5-7c6c3a239bda)

You guys are all ass holes? The screenshot above is the first thing you see when you enter a repository? Are you her to help or just fight foe the throne?

**Comment by LockNessMonster1** [comment]
You´re right @laLocaLuka but this @The-Great-Sephiroth  just behaves has his the greatest guy in the world, has if everything he has done on is life was the true and should be followed by every one! I'm sorry if I cannot agree with that! You don't have to make every project open source, but then, you don't have to share them with the community also!!!

You either keep your projects to your company which must have their self managing rules about clients ans etc, or you keep the project to yourself keeping it private and you're the only one that enjoys it!!! Or you SHARE your project with the community, you don't have to sell you're soul for this, you just make your child available for other to judge, you don't have to accept their suggestions but you will clearly be exposed to much more criticism and suggestions!

Most people are not open to this, I understand, most people think they can control everything in they're lives until they don't!!!

BTW @The-Great-Sephiroth can you point to one of you're apps that is still being widely used today? Since you made so many apps people enjoyed!

As a counterpoint I can point you to thousands of open source projects who thrive on sharing they're code on platforms like this! Heck!!! GitHub is based on GIT which is one of the most successful open source projects in the latest years, Git has grown so big that it couldn't be the success it is without the community, which tests, reports and makes sure it works as it is supposed to work!

Learn from the great!!!!
